arroyo/processing/processor.py

```diff
@@ -145,6 +146,8 @@ def __init__(
 
         self.__commit_policy_state = commit_policy.get_state_machine()
         self.__join_timeout = join_timeout
+        self.__healthcheck_file = healthcheck_file
+        self.__healthcheck_file_touched_at: Optional[float] = None
```

Comment:
Can you revert the changes on this file?

Reply:
whoops